js/script.js

```diff
@@ -1,18 +1,99 @@
-$(function () {
-    //Add active class to nav links
-    $(".navlinks").click(function (){
-      $(".navlinks").toggleClass("active");
-    })
-});
-
-function backgroundChange1() {
-  document.getElementById('top_bg').style.backgroundImage = "url('img/360bg.jpg')";
-}
+// $(function() {
+//   //Add active class to nav links
+//   $('.navlinks').click(function() {
+//     $('.navlinks').toggleClass('active')
+//   })
+// })
 
-function backgroundChange2() {
-  document.getElementById('top_bg').style.backgroundImage = "url('img/360bg.png')";
+function addActiveClass(idName) {
+  document.getElementById(idName).classList.add('active');
 }
 
-function backgroundChange3() {
-  document.getElementById('top_bg').style.backgroundImage = "url('img/360bg.jpg')";
-}
+// if you want to use jQuery:
+// $(window).onload(function(){switch ~~~ })
+window.addEventListener('load', function() {
+  switch (location.pathname) {
+    case '/cafe-menu.php':
+      // if you want to use jQuery:
+      // $('#nav_cafemenu').addClass('active');
+      addActiveClass('nav_cafemenu');
+      break;
+    case '/bar-menu.php':
+      addActiveClass('nav_barmenu');
+      break;
+    case '/photo.php':
+      addActiveClass('nav_photo');
+      break;
+    case '/access.php':
+      addActiveClass('nav_access');
+      break;
+    default:
+      break;
+  }
+})
+
+$(window).on('load', function(){
+  var ua = ua = navigator.userAgent.toLowerCase();
+  var isSP = /iphone|ipod|ipad|android/.test(ua);
+  var count = 0;
+  var timer;
+  var eventStart = isSP ? 'touchstart' : 'mousedown';
+  var eventEnd   = isSP ? 'touchend' : 'mouseup';
+  var eventLeave = isSP ? 'touchmove' : 'mouseleave';
+
+  // rollToLeft
+  document.getElementById('rollToLeft').addEventListener(eventStart, e => {
+    e.preventDefault();
+    var currentPosition = parseInt($("#top_bg").css("background-position-x").replace(/%/, ""));
+    timer = setInterval(function(){
+      count++;
+      $('#top_bg').animate({"background-position-x": (currentPosition + (count * 10)) + "%"}, 0.0001);
+    }, 10)
```

Comment:
`currentPositon` is current the value of `background-position-x`.

By `setInterval` method, you can execute some program at a certain interval.
the second argument of `setInterval` is the interval time(millisecond).
If you give bigger number, roll speed is slower.
Here, `count` is incremented by 1 in 10ms.

jQuery's `animate` method add a certain style to a HTML Element in time of second argument(millisecond).
If you make the number of the second argument, it takes more time to stop background roll.

`count * 10` decides roll speed, too.
if you set a bigger number than `10`, roll speed is faster.